src/revolve/source_template/schemas.py

```
@@ -0,0 +1,8 @@
+import falcon
+class SchemasResource:
```

Pylint (3.3.7)

[refactor] Too few public methods (1/2)

(R0903)